socorro/external/postgresql/staticdata.py

```diff
@@ -100,14 +100,37 @@ class OSVersions(BaseTable):
 class ReleaseRepositories(BaseTable):
     table = 'release_repositories'
     columns = ['repository']
-    rows = [['dev'],
-            ['beta'],
-            ['release'],
-            ['esr'],
-            ['other-dev'],
-            ['other-beta'],
-            ['other-release'],
-            ['other-esr']]
+    rows = [
+        ['mozilla-central'],
+        ['mozilla-1.9.2'],
+        ['comm-central'],
+        ['comm-1.9.2'],
+        ['comm-central-trunk'],
+        ['mozilla-central-android'],
+        ['mozilla-release'],
+        ['mozilla-beta'],
+        ['mozilla-aurora'],
```

Comment:
I want this to be -prod like. I'm not wildly excited about removing things here that are in -prod. When we remove the aurora stuff in bug #1385285 and assuming that covers references in the db, too, I'd be game for removing it in staticdata. Until then, I want to keep it.